source/includes/APIReference/CustomFields/_create.md.erb

```diff
@@ -0,0 +1,40 @@
+## Create Custom Fields
+
+<%= partial "includes/APIReference/CustomFields/Examples/create.tmpl.erb" %>
+
+Add one or more customfields.
+
+### HTTP Request
+
+<img src="images/post.png" alt="post"/><api>https://rest.tsheets.com/api/v1/customfields</api>
+
+### Properties
+_Pass an array of customfield objects as the value to a 'data' property (see example)._
+
+|                |             |             |
+| -------------: | :---------: | ----------- |
+| **name**<br/>required | _String_ | The name of this customfield. |
+| **required**<br/>optional | _Boolean_ | If true, a non-null value must be selected for this customfield when it appears on a timesheet. |
```

Comment:
Could you include what the default value is for any of these optional parameters are?  So that the consumer will know what'll happen if they omit it.

Reply:
Seconding this idea, I can get this data for you if needed.

Reply:
I've added default values now